LEGO1/omni/src/common/mxcompositepresenter.cpp

```diff
@@ -25,7 +25,7 @@ MxCompositePresenter::~MxCompositePresenter()
 }
 
 // FUNCTION: LEGO1 0x100b6410
-// FUNCTION: BETA10 0x100e9d37
+// FUNCTION: BETA10 0x10137344
```

Comment:
This function's `AUTOLOCK` had the wrong filename. However, I found better matching candidates, so this function was likely mismatched.